hydra/hydra.go

```diff
@@ -24,8 +27,10 @@ import (
 	"github.com/libp2p/hydra-booster/metrics"
 	"github.com/libp2p/hydra-booster/periodictasks"
 	"github.com/multiformats/go-multiaddr"
+	mafmt "github.com/multiformats/go-multiaddr-fmt"
 	"go.opencensus.io/stats"
 	"go.opencensus.io/tag"
+	"go.uber.org/atomic"
```

Comment:
`sync/atomic`? I don't think we need to pull in another external dependency. `atomic.AddUint32` should be easy enough?

Reply:
Agreed.